quarkus/config-api/src/main/java/org/keycloak/config/database/Database.java

```diff
@@ -213,4 +257,18 @@ public String toString() {
             return databaseKind.toLowerCase(Locale.ROOT);
         }
     }
+
+    private static class Enabled {
```

Comment:
The intention was to make easy use of methods `isEnabledOnLegacyStore` and `isEnabledOnNewStore` on filters. E.g.:

https://github.com/vramik/keycloak/blob/d760d18dbec0317f48bd43a8c9b6a775384e2dcd/quarkus/config-api/src/main/java/org/keycloak/config/database/Database.java#L115

Reply:
Ok. It's just a nitpick and should not block this PR.

Reply:
@vramik I believe even with an `Enum` this would be achievable, if I'm not mistaken, so @mabartos has a valid point. But it's just a nitpick, non-blocking from my side.